name_node/controllers/outer/replication_controller.go

```diff
@@ -0,0 +1,75 @@
+package outer
```

Comment:
I believe it should be in inner package, but I worked currently with http request, we can move it to inner package and switch to gRPC